mojaloop-technical-overview/account-lookup-service/assets/diagrams/sequence/seq-acct-lookup-post-participants-7.2.1.plantuml

```diff
@@ -96,22 +97,21 @@ group Post Participant's FSP Details
           "currency": "string"
         }
     end note
-    PAYER_FSP --> ALS_API: Request to add participant's FSP details POST - /participants. \n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
+    PAYER_FSP ->> ALS_API: Request to add participant's FSP details\nPOST - /participants\n<color #388724><b>Response code:</b> 202</color> \n<color #FF0000><b>Error code:</b> 200x, 300x, 310x, 320x</color>
```

Comment:
We use '-->' to denote an async callback response in our other sequence diagrams do we not?